permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt

```diff
@@ -63,6 +65,9 @@ actual class PermissionsController(
     }
 
     actual fun isPermissionGranted(permission: Permission): Boolean {
+        if (permission == Permission.REMOTE_NOTIFICATION && Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+            return NotificationManagerCompat.from(applicationContext).areNotificationsEnabled()
+        }
```

Comment:
i think in this case we should also add ability to request from user enabling of notifications. what do you think?

Reply:
on android we don't have to request the Push Notification permission is a _Normal Category Permission_

> If an app declares in its manifest that it needs a normal permission, the system automatically grants the app that permission at install time. The system doesn't prompt the user to grant normal permissions, and users cannot revoke these permissions.

but we have settings that allow to disable notifications

here a situation arises when the request processes can be different on platforms